networkx/drawing/nx_pylab.py

```diff
@@ -638,6 +639,86 @@ def draw_networkx_edges(
     draw_networkx_edge_labels
 
     """
+
+    if arrowstyle == None:
+        if G.is_directed():
+            arrowstyle = "-|>"
+        else:
+            arrowstyle = "-"
+
+    if type(G) == nx.classes.multigraph.MultiGraph == False:
+        _draw_networkx_edges(
```

Comment:
```suggestion
        return _draw_networkx_edges(
```
and similarly for the multigraph drawing routine.

Reply:
`draw_networkx_multigraph_edges` calls `_draw_networkx_edges `method multiple times. I am not sure how to return drawn object in that case.